code/modules/mob/living/simple_animal/abnormality/waw/parasite_tree.dm

```diff
@@ -0,0 +1,382 @@
+	icon = 'ModularTegustation/Teguicons/128x128.dmi'
+	icon_state = "parasitetreeshine"
+	icon_living = "parasitetreeshine"
+	pixel_x = -48
+	base_pixel_x = -48
+	pixel_y = -10
+	base_pixel_y = -10
+	start_qliphoth = 5
+	wander = 0
+	can_patrol = FALSE
+	threat_level = WAW_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 45,
+		ABNORMALITY_WORK_INSIGHT = list(40, 40, 40, 45, 45),
+		ABNORMALITY_WORK_ATTACHMENT = list(50, 50, 50, 50, 55),
+		ABNORMALITY_WORK_REPRESSION = 20
+		)
+	work_damage_amount = 12
+	work_damage_type = WHITE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/hypocrisy,
+		/datum/ego_datum/armor/hypocrisy
+		)
+
+	gift_type =  /datum/ego_gifts/hypocrisy
+	gift_message = "Your ears seem to have grown longer. Weird."
+	var/origin_cooldown = 0 //null when compared to numbers is a eldritch concept so world.time cannot be more or less.
+	var/static/list/blessed = list() //keeps track of status effected individuals
+	var/static/list/minions = list() //keeps track of minions if suppressed forcefully
+
+/mob/living/simple_animal/hostile/abnormality/parasite_tree/Initialize()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_ABNORMALITY_BREACH, .proc/dropLeaf)
+
+/mob/living/simple_animal/hostile/abnormality/parasite_tree/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time, canceled)
+	if(work_type != ABNORMALITY_WORK_REPRESSION && user.stat != DEAD)
+		if(user.has_status_effect(THE_LIARS_BLESSING))
+			var/mob/living/carbon/human/witness = locate(/mob/living/carbon/human) in livinginview(1, src) //included mostly for lore since you encourage people to use it.
+			if(witness && !witness.has_status_effect(THE_LIARS_BLESSING))
+				witness.apply_status_effect(THE_LIARS_BLESSING)
+				to_chat(witness, "<span class='notice'>The air around [src] makes you feel at peace.</span>")
+		else if(datum_reference.qliphoth_meter <= 0)
+			user.apply_status_effect(THE_TREE_CURSE)
+		else
+			user.apply_status_effect(THE_LIARS_BLESSING)
+			if(prob(5))
+				to_chat(user, "<span class='notice'>You hear a voice telling you to bring anyone who needs help here.</span>")
+	else if(work_type == ABNORMALITY_WORK_REPRESSION && datum_reference.qliphoth_meter <= 4)
+		if(canceled || pe < datum_reference.neutral_boxes)
+			to_chat(user, "<span class='notice'>You feel refreshed after being near [src].</span>")
+			if(datum_reference.qliphoth_meter >= 1)
+				user.apply_status_effect(THE_LIARS_BLESSING)
+			else
+				user.apply_status_effect(THE_TREE_CURSE)
+			return ..()
+		else if(locate(THE_TREE_CURSE) in blessed)
+			resetQliphoth()
+			for(var/datum/status_effect/parasite_tree_curse/curse in blessed)
+				qdel(curse)
+			to_chat(user, "<span class='nicegreen'>The scarlet red eye closes as you smash apart [src]'s flowers.</span>")
+			return ..()
+		to_chat(user, "<span class='notice'>[src] stands silently as you finish destroying the buds.</span>")
+		resetQliphoth()
+	return ..()
```

Comment:
```suggestion
/mob/living/simple_animal/hostile/abnormality/parasite_tree/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time, canceled)
	if(work_type != ABNORMALITY_WORK_REPRESSION && user.stat != DEAD)
		if(user.has_status_effect(THE_LIARS_BLESSING))
			var/mob/living/carbon/human/witness = locate(/mob/living/carbon/human) in livinginview(1, src) //included mostly for lore since you encourage people to use it.
			if(witness && !witness.has_status_effect(THE_LIARS_BLESSING))
				witness.apply_status_effect(THE_LIARS_BLESSING)
				to_chat(witness, "<span class='notice'>The air around [src] makes you feel at peace.</span>")
		else if(datum_reference.qliphoth_meter <= 0)
			user.apply_status_effect(THE_TREE_CURSE)
		else
			user.apply_status_effect(THE_LIARS_BLESSING)
			if(prob(5))
				to_chat(user, "<span class='notice'>You hear a voice telling you to bring anyone who needs help here.</span>")
		return
	if(datum_reference.qliphoth_meter <= 4)
		if(canceled || pe < datum_reference.neutral_boxes)
			to_chat(user, "<span class='notice'>You feel refreshed after being near [src].</span>")
			if(datum_reference.qliphoth_meter >= 1)
				user.apply_status_effect(THE_LIARS_BLESSING)
			else
				user.apply_status_effect(THE_TREE_CURSE)
			return ..()
		if(locate(THE_TREE_CURSE) in blessed)
			resetQliphoth()
			for(var/datum/status_effect/parasite_tree_curse/curse in blessed)
				qdel(curse)
			to_chat(user, "<span class='nicegreen'>The scarlet red eye closes as you smash apart [src]'s flowers.</span>")
			return ..()
		to_chat(user, "<span class='notice'>[src] stands silently as you finish destroying the buds.</span>")
		resetQliphoth()
	return ..()
```